### PR TITLE
Added ability to fetch IMDB rating

### PR DESCRIPTION
**Pull request by ahmedsadman**
Based on #260,
Now the Movie module can show both IMDB and TMDB ratings.

### CODE REVIEW

**Comment by ahmedsadman** [comment]
The check failed because I imported a new module named "imdb". How can I fix this?

File: modules/src/movie.py [hunk 1]
```diff
@@ -33,9 +36,16 @@ def process(input, entities):
             })
             data = r.json()
 
+        # fetch the rating from IMDB
+        imdb_id = data['imdb_id']
+        imdb_search = ia.search_movie(imdb_id)
```
Comment (swapagarwal) [inline modules/src/movie.py]: The `search_movie` method is useful when we are querying the movie using keywords. Here, we already have the IMDb ID. We can directly use the `get_movie` method here.

File: modules/src/movie.py [hunk 2]
```diff
@@ -33,9 +36,16 @@ def process(input, entities):
...
+        imdb_search = imdb_search[0]
+        ia.update(imdb_search)
+
         template = TextTemplate('Title: ' + data['title'] +
                                 '\nYear: ' + data['release_date'][:4] +
-                                '\nAverage Rating: ' + str(data['vote_average']) + ' / 10' +
+                                '\nTMDB Rating: ' + str(data['vote_average']) + ' / 10' +
```
Comment (swapagarwal) [inline modules/src/movie.py]: Let's keep only one rating (imdb) here.

File: modules/src/movie.py (hunk 1 above)
Comment (ahmedsadman) [inline modules/src/movie.py]: Done. I didn't know about get_movie.

File: modules/src/movie.py (hunk 2 above)
Comment (ahmedsadman) [inline modules/src/movie.py]: Done

**Comment by swapagarwal** [comment]
Thanks @ahmedsadman! 👍

**Pull request merged**